tests/console/yast2_cmdline.pm

```diff
@@ -40,8 +42,22 @@ sub run_yast_cli_test {
 }
 
 sub run {
+    my $cmd;
+
     select_console 'root-console';
 
+    if (is_sle) {
+        if (sle_version_at_least('15') and get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE')) {
```

Comment:
I guess we should extract this to common method which will be called from here and zypper_info to follow DRY

Reply:
where else is it called?

btw, better use the new function `is_sle('15+')` please

Reply:
Agreed, but where to put it ? What do you suggest ? utils.pm ?

Reply:
I have got it from zypper_info.

Reply:
At least for the name, I believe lib/repo_tools.pm should fit the purpose, as of now it's used only for smt/rmt.

Reply:
Oli, wdyt about Rodion's proposal ?

Reply:
@okurz, could you please comment here? I guess it's last thing we have to do before merging.